manifests/Intel/Iometer/1.1.0.yaml

```diff
@@ -0,0 +1,17 @@
+Id: Intel.Iometer
+Publisher: Intel
+Author: Intel
+Name: Iometer
+Version: 1.1.0
+License: Intel Open Source License
+InstallerType: EXE
+Tags: network, benchmark
+Description: As the Iometer User's Guide says, Iometer is an I/O subsystem measurement and characterization tool for single and clustered systems. It was originally developed by the Intel Corporation and announced at the Intel Developers Forum (IDF) on February 17, 1998 - since then it got wide spread within the industry.
+Homepage: https://www.snort.org/
```

Comment:
Please provide link to the iometer homepage. It would also be helpful to have a link to the LicenseUrl. Thanks!